tools/clang/test/CodeGenDXIL/hlsl/shaders/node/mesh-node-writable-input-record.hlsl

```diff
@@ -0,0 +1,23 @@
+// RUN: %dxc -Tlib_6_9 %s -verify 
+
+// Make sure writable input records aren't allowed for mesh node shaders
```

Comment:
The issue isn't writable input records, but Group and Thread input records meant for coalescing and thread launch only.  Should have an entry for each of those objects used as inputs (`[RW](Group|Thread)NodeInputRecord(s)`), which could all be in the same test.